internal/commands/suggest_stacks_test.go

```diff
@@ -31,7 +31,8 @@ func testSuggestStacksCommand(t *testing.T, when spec.G, it spec.S) {
 		it("displays stack information", func() {
 			command.SetArgs([]string{})
 			h.AssertNil(t, command.Execute())
-			h.AssertEq(t, outBuf.String(), `Stacks maintained by the community:
+			h.AssertEq(t, outBuf.String(), "\x1b\x5b\x33\x33\x3b\x31\x6dWarning: \x1b\x5b\x30\x6dCommand 'pack suggest-stacks' has been deprecated, please use 'pack stack suggest' instead\n"+
```

Comment:
nit: Instead of having the codes, add 
```
	color.Disable(true)
	defer color.Disable(false)
```
to `TestSuggestStacksCommand`, like we do here: https://github.com/buildpacks/pack/blob/1efc582d28151f115f874f5a5372aad1e833ffd2/internal/commands/set_default_builder_test.go#L25-L29